portality/api/v1/crud/applications.py

```diff
@@ -109,7 +110,7 @@ def update(cls, id, data, account):
 
         # finally save the new application, and return to the caller
         new_ap.save()
-        return ap
+        return new_ap
```

Comment:
Yes, my next PR for the API is going to be another look at Api tests - I'll add that to my list.